Tools/plutil/main.swift

```diff
@@ -9,7 +9,7 @@
 #if os(macOS) || os(iOS)
 import Darwin
 import SwiftFoundation
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
 import Foundation
```

Comment:
Might be better to rewrite this to use `#if canImport(Darwin)...` and `#if canImport(Glibc)...`